src/main/java/org/jenkinsci/plugins/pipeline/milestone/Milestone.java

```diff
@@ -59,7 +57,6 @@ class Milestone {
     }
 
     public void pass(StepContext context, Run<?, ?> build) {
```

Comment:
We could just delete any methods and even make the constructor throw `AssertionError`, right? So that only its serial form remains?